cdap-ui/BUILD.rst

```diff
@@ -0,0 +1,118 @@
+=======
+CDAP UI
+=======
+
+CDAP UI comprises three different webapps -- CDAP, Hydrator, and Tracker -- that we ship as part of every release.
+CDAP is built in React while Hydrator and Tracker are written in Angular.
+
+Building the UI
+===============
+
+Prerequisites
+-------------
+- NodeJS Version: 4.5.0 or higher
+
+  CDAP UI requires a minimum NodeJS version of 4.5.0.
+  You could either download from the nodejs.org website or use a version manager.
+
+  - `v4.5.0 Download <https://nodejs.org/download/release/v4.5.0/>`__
```

Comment:
I just renamed this file. Can we fix it in a follow up? Otherwise give me the instructions, and I'm happy to do it